src/game/client/neo/ui/neo_root_settings.cpp

```diff
@@ -969,6 +982,8 @@ void NeoSettings_Crosshair(NeoSettings *ns)
 			NeoUI::SliderInt(L"Circle radius", &pCrosshair->info.iCircleRad, 0, CROSSHAIR_MAX_CIRCLE_RAD);
 			NeoUI::SliderInt(L"Circle segments", &pCrosshair->info.iCircleSegments, 0, CROSSHAIR_MAX_CIRCLE_SEGMENTS);
 		}
+		NeoUI::HeadingLabel(L"NETWORKING");
+		NeoUI::RingBoxBool(L"Show other players crosshair", &pCrosshair->bNetworkCrosshair);
```

Comment:
Language nit: Maybe use the plural possessive, so `players' crosshairs`

Same with the description text at cvar `cl_neo_crosshair_network` declaration in file `src/game/client/neo/ui/neo_hud_crosshair.cpp`